.kokoro/tests/run_single_test.sh

```diff
@@ -50,20 +50,36 @@ if [[ "${INJECT_REGION_TAGS:-}" == "true" ]]; then
     export XUNIT_PATH="$PWD/sponge_log.xml"
     export XUNIT_TMP_PATH="$(mktemp)"
 
+    # We use `python3.8` because it is the version pip3 installs to.
+    export PYTHON_CMD="python3.8"
```

Comment:
Is it okay to assume this @tmatsuo @kurtisvg?

Reply:
N.B: If not, we'll have to find some other way to have `pip` target a different version.

(We could manually add to `PYTHONPATH`, but that feels kludgey IMO.)

Reply:
If possible, I'd like to just use major version e.g. `python3`.

Reply:
I will add some commits to use `python3` instead of using the specific version.